HotTips/TipOfTheDayWindow.xaml.cs

```diff
@@ -308,6 +332,7 @@ private void PopulateLikeImage()
 
         private void PopulateDislikeFilledImage()
         {
+            currentTip.tipLikeStatus = TipLikeEnum.DISLIKE;
```

Comment:
Not sure why this would be done here. Seems like the wrong place to be setting current tip value.